response_test.go

```diff
@@ -1674,3 +1690,152 @@ func TestResponse_Usage(t *testing.T) {
 		resp.chain.assert(t, failure)
 	})
 }
+
+func TestResponse_Reader(t *testing.T) {
+	t.Run("get reader", func(t *testing.T) {
```

Comment:
In this test, can we also check that we can actually read body from returned reader, without errors, and get expected string?